### PR TITLE
ST-1329-sptribs-ithc and perftest image policy

### PR DESCRIPTION
**Pull request by geciciserkan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-1329

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by hannah38 — APPROVED** [review]

**Review by timja — APPROVED** [review]

File: apps/sptribs/sptribs-case-api/perftest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
```
Comment (hannah38) [inline apps/sptribs/sptribs-case-api/perftest.yaml:1]:
```suggestion
apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

File: apps/sptribs/sptribs-case-api/perftest.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+kind: ImageRepository
```
Comment (hannah38) [inline apps/sptribs/sptribs-case-api/perftest.yaml:2]:
```suggestion
kind: HelmRelease
```

File: apps/sptribs/sptribs-case-api/ithc.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
```
Comment (hannah38) [inline apps/sptribs/sptribs-case-api/ithc.yaml:1]:
```suggestion
apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

**Review by hannah38 — CHANGES_REQUESTED** [review]
Hey, the patches are meant to be on the HelmRelease so I've asked for a few changes there

**Review by hannah38 — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**